src/App.js

```diff
@@ -1,7 +1,11 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+import './App.css';
+
 function App() {
```

Comment:
- [ ] It would be better if your `App` component is class-based as one of the requirements. Kindly fix this minor issue. :+1: :smile: